src/styles/styles.js

```diff
@@ -266,6 +266,11 @@ const styles = {
         borderWidth: 0,
     },
 
+    buttonDisable: {
+        backgroundColor: themeColors.textMutedReversed,
```

Comment:
Can you actually create a new `themeColor` for this that uses `colors.gray2` please Perhaps name it `themeColors.buttonDisabledBG`

Reply:
@shanky97

Reply:
@marcaaron  @shawnborton  I have updated the changes, Please check.